src/coreclr/jit/targetwasm.h

```diff
@@ -44,7 +44,11 @@
   #define FEATURE_EH_CALLFINALLY_THUNKS 1  // Generate call-to-finally code in "thunks" in the enclosing EH region, protected by "cloned finally" clauses.
   #define CSE_CONSTS               1       // Enable if we want to CSE constants
 
-  #define RBM_ALLFLOAT             RBM_F0
+  #define RBM_R0                   REG_NA
+  #define RBM_F0                   REG_NA
+  #define REG_F0                   REG_NA
```

Comment:
`RBM_<...>` is a register mask, i. e. `1 << REG_<...>`. So the right "nothing" value for it is zero, i. e. `RBM_NONE`.
```suggestion
  #define RBM_R0                   RBM_NONE
  #define RBM_F0                   RBM_NONE
  #define REG_F0                   REG_NA
```